client/my-sites/stats/activity-log-day/index.jsx

```diff
@@ -116,20 +119,25 @@ class ActivityLogDay extends Component {
 		return (
 			<div>
 				<div className="activity-log-day__day">
+					{ /* eslint-disable indent */ }
 					{ isToday
 						? translate( '%s — Today', {
-							args: formattedDate,
-							comment: 'Long date with today indicator, i.e. "January 1, 2017 — Today"',
-						} )
-						: formattedDate
-					}
+								args: formattedDate,
+								comment: 'Long date with today indicator, i.e. "January 1, 2017 — Today"',
+							} )
+						: formattedDate }
+					{ /* eslint-enable indent */ }
+				</div>
+				<div className="activity-log-day__events">
+					{ /* eslint-disable indent */ }
+					{ isEmpty( logs )
+						? translate( 'No activity' )
```

Comment:
Hi! I've found a possible matching string that has already been translated **84** times:
`translate( 'No activity yet!' )` *ES Score: 12*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/activity-log/empty-days)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).